src/compositions/Timeline/Timeline.css

```
@@ -0,0 +1,56 @@
+.statusTimeline {
+  margin-top: 50px;
+}
+
+.statusTimelineUl {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  padding: 0;
+  position: relative;
+
+  &:before {
+    position: absolute;
+    content: " ";
+    background-color: #c7d9f6;
+    height: 2px;
+    top: 2px;
+    bottom: 26px;
+    left: 30px;
+    right: 0;
+    width: calc(100% - 80px);
+    margin-bottom: 20px;
+  }
+}
```

```suggestion
.statusTimelineUl {
  display: flex;
  list-style-type: none;
  justify-content: space-between;
  padding: 0;
  position: relative;
}
 .statusTimelineUl:before {
    position: absolute;
    content: " ";
    background-color: #c7d9f6;
    height: 2px;
    top: 2px;
    bottom: 26px;
    left: 30px;
    right: 0;
    width: calc(100% - 80px);
    margin-bottom: 20px;
}
```

Correction in removing nesting -